wampy/transports/websocket/frames.py

```diff
@@ -115,6 +115,9 @@ def __init__(self, bytes):
         self.opcode = self.OPCODE_TEXT
         self.payload = self.generate_payload()
 
+    def dataToBytes(self, data):
```

Comment:
a style guide would be useful i think. sorry to nag again about style, but wampy prefers `data_to_bytes`. I'll try and find time to put these style preferences in a wiki page or contributing file.